package.json

```diff
@@ -19,7 +19,9 @@
     "test-v:better": "forge test -vvvv --etherscan-api-key $(grep ETHERSCAN_API_KEY .env | cut -d '=' -f2)",
     "size": "forge build --sizes --force --via-ir",
     "compile": "forge build",
-    "deploy:mainnet": "forge script src/script/MainnetDeployment.s.sol:MainnetDeployment --rpc-url $(grep MAINNET_RPC .env | cut -d '=' -f2) --private-key $(grep PRIVATE_KEY .env | cut -d '=' -f2) --broadcast --verify --etherscan-api-key $(grep ETHERSCAN_API_KEY .env | cut -d '=' -f2) -vvvv",
+    "deploy:mainnet:uniswap": "forge script src/script/uniswap/UniswapMainnetDeployment.s.sol:UniswapMainnetDeployment --rpc-url $(grep MAINNET_RPC .env | cut -d '=' -f2) --private-key $(grep PRIVATE_KEY .env | cut -d '=' -f2) --broadcast --verify --etherscan-api-key $(grep MAINNET_ETHERSCAN_API_KEY .env | cut -d '=' -f2) -vvvv",
```

Comment:
This doesn't allow to run a foundry simulation, I think the following would be better:
```shell
yarn deploy:mainnet:uniswap
yarn deploy:mainnet:uniswap --broadcast
yarn deploy:mainnet:uniswap --broadcast --verify
yarn deploy:mainnet:uniswap --broadcast --verify --verifier sourcify
```
So, let's remove `--broadcast --verify` from here.